include/magma/vk/buffers/BufferManager.hpp

```diff
@@ -0,0 +1,40 @@
+#include "magma/vk/buffers/BufferManager.h"
```

Comment:
this include should be deleted.
It creates recursive include problem

Reply:
Also there should be #pragma once

Reply:
Done